platform/app/cypress/integration/customization/OHIFDoubleClick.spec.js

```diff
@@ -16,12 +16,18 @@ describe('OHIF Double Click', () => {
       .should('be.eq', numExpectedViewports);
 
     for (let i = 0; i < numExpectedViewports; i += 1) {
+      cy.wait(2000);
```

Comment:
If you merge the CINE fix in CS3D, then this wait gets removed and you just do a waitDicomImage() above line 18.
I'd really like to see it do waitDicomImage instead of cy.wait as it is much more reliable, and is fast on fast systems, and slow on slower connections.